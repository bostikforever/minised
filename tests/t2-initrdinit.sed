/file system/!d
s/file system.*//
s/ //g
y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/
