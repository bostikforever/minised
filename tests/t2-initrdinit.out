hfs
hfsplus
reiserfs
ext3
